src/test/java/GmailClientTest.java

```diff
@@ -119,14 +124,31 @@ public void getQueryStringFromAnySender() {
     Assert.assertEquals(fromAnySenderQuery, EMPTY_QUERY);
   }
 
+  @Test
+  public void getQueryStringImportantEmailsOnly() {
+    String importantQuery = GmailClient.isImportantQuery(RETURN_IMPORTANT_ONLY);
+
+    Assert.assertEquals(importantQuery, IMPORTANT_EMAILS_QUERY);
+  }
+
+  @Test
+  public void getQueryStringDontFilterForImportantTag() {
+    String noImportantFilterQuery = GmailClient.isImportantQuery(DEFAULT_IMPORTANT_FILTER);
```

Comment:
Nit: I think `false` would be clearer

Reply:
Just for consistency, I think it might be better to continue using constants for every value (we agreed to this in a standup a while back). 

I don't mind changing this though if you have strong feelings either way. Please let me know!

Reply:
Fair enough - for booleans I'll just use the values.